### PR TITLE
Bert Haine: x gon give it to ya

### PR DESCRIPTION
**Pull request by BertHaine**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 3

Completeness Level (1-5): 4

What did you think of this deliverable?: I had a lot of difficulty with the game logic and eventually called on a friend to help walk me through much of it. Once I was finished with that I had fun styling it while listening to DMX.

### CODE REVIEW

**Review by Jacob-Elder — COMMENTED** [review]
- Code is very efficient! :)
- The event parameter doesn't appear to be necessary in your newGame click event.
- For your html, put your script tag inside of your closing body tag. Supposedly IE wouldn't run your script if its outside the closing body tag.

**Review by SarahPappas — COMMENTED** [review]
Really nice and simple. Great winning logic function.

JS -
     - In your click events to run game logic, I think alerts are run with "window.alert(ALERT_CONTENT)".
     - In your click events to run game logic you add a class of “true”… I didn’t see this class in your JS. It would be helpful, if you’re only using the class for JS to label it to somehow indicate that, like “js_true".
     - In your function “isWinner” maybe should be named “checkWinner” isWinner suggests the function will return a boolean.

**Review by tannerkovach — COMMENTED** [review]
Well organized and nice and short javascript file. Also the use of images was an awesome touch. Plus any reference to DMX wins automatically.

File: js/app.js [hunk 1]
```diff
@@ -1 +1,74 @@
...
+	});	
+}
+
+
+//identify winning combos
+var winningNumbers = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [2, 4, 6], [0, 4, 8]];
+
+
+//identify winner
+var isWinner = function() {
```
Comment (brandiw) [inline js/app.js:54]: Interesting way to check winner - I hadn't thought of it!

**Review by brandiw — COMMENTED** [review]
Creative implementation!

**Pull request closed without merging**